fontforge/sfd.c

```diff
@@ -764,6 +770,7 @@ static void SFDDumpSplineSet(FILE *sfd,SplineSet *spl) {
 	    fprintf( sfd, "  PathStart: %d\n", spl->start_offset );
 	}
     }
+    if (order2) free(nspl);
```

Comment:
Are you sure that this is in the right block?

Reply:
@frank-trampe Sorry, fixed in a3aff8a6e1390a3fdee8b318a28d6b8f47a2211d

Reply:
Unresolving: nspl is a linked list that needs to be freed with `SplinePointListFree()` (SplinePointList and SplineSet have the same typedef I believe.)